service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/preselector/ManageOrderInitialTypePreSelectorTest.java

```diff
@@ -16,10 +15,10 @@
 import static uk.gov.hmcts.reform.fpl.enums.State.CLOSED;
 import static uk.gov.hmcts.reform.fpl.model.order.Order.C21_BLANK_ORDER;
 import static uk.gov.hmcts.reform.fpl.model.order.OrderSection.ISSUING_DETAILS;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.emptyCaseDetails;
 
 class ManageOrderInitialTypePreSelectorTest {
 
-    private static final CaseDetails CASE_DETAILS = CaseDetails.builder().data(new HashMap<>()).build();
```

Comment:
static filed that are mutated during test keep their state between test execution